mkosi/__init__.py

```diff
@@ -2447,10 +2477,27 @@ def invoke_apt(
         INITRD="No",
     )
 
-    with mount_api_vfs(args, root):
+    with mount_api_vfs(args, root), mount_apt_local_mirror(args, root), mount_apt_override_config(root):
```

Comment:
Do we need to mount the local mirror everytime we invoke apt or is it sufficient to do it for the "install" operation?

Reply:
yes, the repository metadata is also in the mirror, so update/search need it too